silkworm/rpc/http/json_rpc_validator.cpp

```diff
@@ -0,0 +1,337 @@
+/*
+   Copyright 2023 The Silkworm Authors
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+
+#include "json_rpc_validator.hpp"
+
+#include <bit>
+#include <cstdlib>
+#include <exception>
+#include <filesystem>
+#include <fstream>
+#include <iostream>
+#include <regex>
+#include <string>
+#include <utility>
+#include <vector>
+
+#include "json_rpc_specification.hpp"
+
+namespace silkworm::rpc::http {
+
+#define REQUEST_FIELD_METHOD "method"
+#define REQUEST_FIELD_ID "id"
+#define REQUEST_FIELD_PARAMETERS "params"
+#define REQUEST_FIELD_JSONRPC "jsonrpc"
```

Comment:
Please use `constexpr` or `const` variables for constants instead of `#define`, as specified in [Google C++ Style Guide](https://google.github.io/styleguide/cppguide.html#Constant_Names). You can do it in next PR.